neard/src/main.rs

```diff
@@ -1,25 +1,41 @@
-use std::convert::TryInto;
+mod init;
+mod run;
+mod testnet;
+
 use std::env;
 use std::fs;
 use std::io;
 use std::path::Path;
 
-use clap::{crate_version, App, AppSettings, Arg, SubCommand};
+use clap::{crate_version, AppSettings, Clap};
 #[cfg(feature = "adversarial")]
 use tracing::error;
 use tracing::info;
 use tracing_subscriber::filter::LevelFilter;
 use tracing_subscriber::EnvFilter;
 
-use actix;
 use git_version::git_version;
 use near_performance_metrics;
 use near_primitives::version::{Version, DB_VERSION, PROTOCOL_VERSION};
 #[cfg(feature = "memory_stats")]
 use near_rust_allocator_proxy::allocator::MyAllocator;
-use nearcore::config::{init_testnet_configs, load_config_without_genesis_records};
-use nearcore::genesis_validate::validate_genesis;
-use nearcore::{get_default_home, get_store_path, init_configs, start_with_config};
+use nearcore::{get_default_home, get_store_path};
+
+#[macro_use]
+extern crate lazy_static;
```

Comment:
Don't use `extern crate` syntax, it is not needed in 2018 edition. Instead, `use lazy_static::lazy_static`.

Reply:
oops!